### ISSUE TITLE
As a Voter, I want the components to be accessible so that I can use the screen Reader and keyboard to navigate through the application.

### ISSUE THREAD

**Issue by jaguarfi** [OpenVAA/voting-advice-application#153]
List of Components that have been developed so far and need to improve the accessibility.

- Card
- Tab
- EntityCard
- ScoreGauge
- Question
- CandidateDetailsCard
- CandidateBasicInfo

And basically, all the other components inside the `lib/components` and `lib/shared` folder

**Resources**
- https://github.com/microsoft/TypeScript/blob/main/src/lib/dom.generated.d.ts
- https://m3.material.io/components/cards/accessibility

**Comment by kaljarv** [comment]
One point to note is that `<EntityCard>` is currently not accessible because tabbing only cycles through the `<ScoreGauge>` components which do not function as links to open the card.

My take would be that instead of the Feed pattern the Grid pattern be used such that:
- the user can use the up/down arrows to move between candidates so that the information for the same 'column' is spoken aloud
- hitting space or enter always opens the current candidate ('row')

E.g.

1. Open the results page
2. Use `tab` to cycle through interactive widgets to the results grid
3. Use `right arrow` to cycle through the fields ('cols') of the first candidate, such as, name, party, match score
4. Settle on match score, and keep hitting `down arrow` to hear the match scores for candidates down the list
5. At any candidate, `left/right arrow` can be used to move to other fields

Optionally, a screen-reader-only summary field can also be added as the first 'col' that includes a combined textual summary of the most important info in the card.

**Comment by kaljarv** [comment]
Reasses the discussion here as some of the changes may have been made already.

Consider the Grid pattern mentioned above.
